app/src/main/res/layout/merge_venue_info_layout.xml

```diff
@@ -41,6 +41,8 @@
     android:id="@+id/venue_map"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:importantForAccessibility="no" />
+    android:foreground="?android:attr/selectableItemBackground"
```

Comment:
❌ this will only work on API 23+, please consider extracting the foreground logic we have already in `CircleImageView` to an `ImageViewWithForeground` and use it here, so that we can do this safely on Lollipop.